stream.html

```diff
@@ -144,6 +153,11 @@
 
 <script>
 
+let ARENA = new URL(window.location.href).searchParams.get('arena');
```

Comment:
nit: why is this in caps if it's a `let` but also why is it a `let` if it doesn't get re-assigned?